integration_tests/mockApis/lostBed.ts

```diff
@@ -21,6 +21,55 @@ export default {
       },
     }),
 
+  stubLostBedUpdate: (args: { lostBed: LostBed; premisesId: string }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'PUT',
+        url: `/premises/${args.premisesId}/lost-beds/${args.lostBed.id}`,
```

Comment:
I would prefer to use the `static-path` paths for these too if it's straightforward to change them. Again, I realise not all the stubs in this file use them but I think new ones should